.github/dependabot.yml

```
@@ -1,96 +1,223 @@
-    directory: "/golang/year-service"
-    <<: *defaults
-# JAVA
-  - package-ecosystem: "gradle"
-    directory: "/java/frontend"
-    <<: *defaults
-  - package-ecosystem: "gradle"
-    directory: "/java/message-service"
-    <<: *defaults
-  - package-ecosystem: "gradle"
-    directory: "/java/name-service"
-    <<: *defaults
-  - package-ecosystem: "gradle"
-    directory: "/java/year-service"
-    <<: *defaults
-# NODE
-  - package-ecosystem: "npm"
-    directory: "/node/frontend"
-    <<: *defaults
-  - package-ecosystem: "npm"
-    directory: "/node/message-service"
-    <<: *defaults
-  - package-ecosystem: "npm"
-    directory: "/node/name-service"
-    <<: *defaults
-  - package-ecosystem: "npm"
-    directory: "/node/year-service"
-    <<: *defaults
-# PYTHON
-  - package-ecosystem: "pip"
-    directory: "/python/frontend"
-    <<: *defaults
-  - package-ecosystem: "pip"
-    directory: "/python/message-service"
-    <<: *defaults
-  - package-ecosystem: "pip"
-    directory: "/python/name-service"
-    <<: *defaults
-  - package-ecosystem: "pip"
-    directory: "/python/year-service"
-    <<: *defaults
-# RUBY
-  - package-ecosystem: "bundler"
-    directory: "/ruby/frontend"
-    <<: *defaults
-  - package-ecosystem: "bundler"
-    directory: "/ruby/message-service"
-    <<: *defaults
-  - package-ecosystem: "bundler"
-    directory: "/ruby/name-service"
-    <<: *defaults
-  - package-ecosystem: "bundler"
-    directory: "/ruby/year-service"
-    <<: *defaults
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+# DOTNET
+  - package-ecosystem: "nuget"
+    directory: "/dotnet"
+    schedule:
+        interval: "monthly"
+    labels:
+        - "type: dependencies"
+    reviewers:
+        - "honeycombio/telemetry-team"
+# ELIXIR
+  - package-ecosystem: "mix"
+    directory: "/elixir/frontend"
+    schedule:
+        interval: "monthly"
+    labels:
+        - "type: dependencies"
+    reviewers:
+        - "honeycombio/telemetry-team"
+  - package-ecosystem: "mix"
+    directory: "/elixir/message"
+    schedule:
+        interval: "monthly"
+    labels:
+        - "type: dependencies"
+    reviewers:
+        - "honeycombio/telemetry-team"
+  - package-ecosystem: "mix"
+    directory: "/elixir/name"
+    schedule:
+        interval: "monthly"
+    labels:
+        - "type: dependencies"
+    reviewers:
+        - "honeycombio/telemetry-team"
+  - package-ecosystem: "mix"
+    directory: "/elixir/year"
+    schedule:
+        interval: "monthly"
+    labels:
+        - "type: dependencies"
+    reviewers:
+        - "honeycombio/telemetry-team"
+# GOLANG
+  - package-ecosystem: "gomod"
+    directory: "/golang/frontend"

```

I think the section for go needs to change to be similar to dotnet now - at some point the individual directories dropped the `go.mod` files and they all use the main one in the `golang` directory.
```suggestion
    directory: "/golang"
```

---

wonderful! less copy past